ddtrace/settings/profiling.py

```diff
@@ -37,6 +39,11 @@ def _derive_default_heap_sample_size(heap_config, default_heap_sample_size=1024
     return int(max(math.ceil(total_mem / max_samples), default_heap_sample_size))
 
 
+def _is_glibc_linux_x86_64():
```

Comment:
it looks like we need this logic in multiple places so perhaps we can make it a util helper?

Reply:
Indeed; we do need it in two places (profiler.py settings and in setup.py) (although this will go away once the universe has submitted to the wonders of libdatadog).  Could you propose a sensible place to consolidate?

Reply:
Ah if we need it in the setup.py script, we would have to duplicate the code there unfortunately. I see this helper occurring also in `ddtrace/profiling/profiler.py`. Perhaps we could make it an `En.d` (a derivation) as part of the profiler configuration
~~~ python
class ProfilerConfig(En):
    ...
    _is_glibc_linux_x86_64 = En.d(
        bool,
        lambda c: _is_glibc_linux_x86_64(),
    )
~~~
and then retrieve it from the `config` object when needed?

Reply:
Ah; apologies--the profiler.py thing is a mistake.  It indeed should all be consolidated within `config` whenever possible.  Will fix--thank you!

Reply:
Fixed the profiler.py thing in d4181b151.  In general, the config should be a single source of truth for this stuff when possible imho.